mlflow/server/js/src/common/utils/Utils.tsx

```diff
@@ -665,6 +666,14 @@ class Utils {
     return '';
   }
 
+  static getRunDescriptionFromTags(runTags: any) {
+    const runDescriptionTag = runTags[Utils.runDescriptionTag];
```

Comment:
instead of defining `runDescriptionTag`, you can reuse `NOTE_CONTENT_TAG` from `mlflow/server/js/src/experiment-tracking/utils/NoteUtils.ts`